### PR TITLE
Add explicit prefixes for each techport

### PR DESCRIPTION
**Pull request by bnaecker**

### CODE REVIEW

File: package-manifest.toml [hunk 1]
```diff
@@ -318,8 +318,8 @@ only_for_targets.image = "standard"
 # 2. Copy dendrite.tar.gz from dendrite/out to omicron/out
 source.type = "prebuilt"
 source.repo = "dendrite"
-source.commit = "6de087190ed4afa7ffbcede51140a7eff4cc5a60"
-source.sha256 = "4576e3e8b5eb83b9fb8d3d867998bd25f6a9f1f30c92868bf745afa46e4e6926"
+source.commit = "51fb973674dfc1c372621bd83970c147684a005f"
```
Comment (bnaecker) [inline package-manifest.toml:321]: Point to commit on `main` once https://github.com/oxidecomputer/dendrite/pull/448 is merged. Ditto for others below.

**Comment by bnaecker** [comment]
Ok, I've finally managed to test this on `madrid`. I built everything locally using `omicron-package package`, where the target is:

```bash
bnaecker@feldspar : ~/omicron $ ./target/release/omicron-package target list
             dogfood: image=standard machine=gimlet switch=asic
             home: image=standard machine=non-gimlet switch=stub
SELECTED >>> madrid: image=standard machine=gimlet switch=asic
```

Note that this is currently using a manually updated version of Dendrite, pointing at commit `51fb973674dfc1c372621bd83970c147684a005f` which has the corresponding changes from https://github.com/oxidecomputer/dendrite/pull/448.

I copied the resulting sled-agent and switch tarballs manually to `madrid`; unpacked them in place; and then imported / started the sled agent SMF service. We can see that starts up as expected:

```bash
BRM44220001 # svcadm enable sled-agent && tail -f $(svcs -L sled-agent)
{"msg":"request completed","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:00:20.168648956Z","hostname":"BRM44220001","pid":670,"uri":"/baseboard","method":"GET","req_id":"e796c311-8772-48e7-8e8f-1d83d25b060c","remote_addr":"[fdb0:a840:2504:355::2]:38485","local_addr":"[fdb0:a840:2504:355::1]:80","component":"dropshot (BootstrapAgent)","response_code":"200"}
{"msg":"accepted connection","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:00:50.179469469Z","hostname":"BRM44220001","pid":670,"local_addr":"[fdb0:a840:2504:355::1]:80","component":"dropshot (BootstrapAgent)","remote_addr":"[fdb0:a840:2504:355::2]:32947"}
{"msg":"request completed","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:00:50.184818659Z","hostname":"BRM44220001","pid":670,"uri":"/baseboard","method":"GET","req_id":"f5c5ecc0-6f69-43ed-80ad-6b0ad40bc942","remote_addr":"[fdb0:a840:2504:355::2]:32947","local_addr":"[fdb0:a840:2504:355::1]:80","component":"dropshot (BootstrapAgent)","response_code":"200"}
[ Jan 10 09:00:55 Stopping because service disabled. ]
[ Jan 10 09:00:55 Executing stop method (:kill). ]
[ Jan 10 09:05:06 Rereading configuration. ]
[ Jan 10 09:05:45 Rereading configuration. ]
[ Jan 10 09:06:14 Enabled. ]
[ Jan 10 09:06:14 Executing start method ("ctrun -l child -o noorphan,regent /opt/oxide/sled-agent/sled-agent run /opt/oxide/sled-agent/pkg/config.toml &"). ]
[ Jan 10 09:06:14 Method "start" exited with status 0. ]
note: configured to log to "/dev/stdout"
{"msg":"Starting mg-ddm service","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:14.696520174Z","hostname":"BRM44220001","pid":7932}
{"msg":"Importing mg-ddm service","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:14.697090291Z","hostname":"BRM44220001","pid":7932,"path":"/opt/oxide/mg-ddm/pkg/ddm/manifest.xml"}
{"msg":"Setting mg-ddm interfaces","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:14.841070922Z","hostname":"BRM44220001","pid":7932,"interfaces":"(\"cxgbe0/ll\" \"cxgbe1/ll\")"}
{"msg":"Enabling mg-ddm service","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:14.856228616Z","hostname":"BRM44220001","pid":7932}
...
```

A few seconds later it starts the switch zone, which starts the `tfport` service:

```bash
{"msg":"Zone booting","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:21.55520392Z","hostname":"BRM44220001","pid":7932,"zone":"oxz_switch","component":"ServiceManager","component":"BootstrapAgent"}
{"msg":"Ensuring bootstrap address fdb0:a840:2504:355::2 exists in switch zone","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:28.042311915Z","hostname":"BRM44220001","pid":7932,"component":"ServiceManager","component":"BootstrapAgent"}
{"msg":"Adding bootstrap address","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:28.042377881Z","hostname":"BRM44220001","pid":7932,"zone":"oxz_switch","component":"ServiceManager","component":"BootstrapAgent"}
{"msg":"Forwarding bootstrap traffic via oxBootstrap0 to fe80::8:20ff:fe99:fbfa","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:38.30013834Z","hostname":"BRM44220001","pid":7932,"component":"ServiceManager","component":"BootstrapAgent"}
{"msg":"GZ addresses: []","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:38.312176656Z","hostname":"BRM44220001","pid":7932,"component":"ServiceManager","component":"BootstrapAgent"}
{"msg":"Zone not using gateway (underlay is not up)","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:38.312228204Z","hostname":"BRM44220001","pid":7932,"component":"ServiceManager","component":"BootstrapAgent"}
{"msg":"Setting up dendrite service","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:38.4767083Z","hostname":"BRM44220001","pid":7932,"component":"ServiceManager","component":"BootstrapAgent"}
{"msg":"no rack_id/sled_id available yet","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:38.476787612Z","hostname":"BRM44220001","pid":7932,"component":"ServiceManager","component":"BootstrapAgent"}
{"msg":"Setting up MGS service","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:38.76734775Z","hostname":"BRM44220001","pid":7932,"component":"ServiceManager","component":"BootstrapAgent"}
{"msg":"Setting up tfport service","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:38.97453204Z","hostname":"BRM44220001","pid":7932,"component":"ServiceManager","component":"BootstrapAgent"}
{"msg":"Setting up wicketd service","v":0,"name":"SledAgent","level":30,"time":"1987-01-10T09:06:39.232365263Z","hostname":"BRM44220001","pid":7932,"component":"ServiceManager","component":"BootstrapAgent"}
```

The `tfport` service itself has the expected prefixes assigned to it:

```bash
BRM44220001 # zlogin oxz_switch svcprop -p config tfport
config/vlan_data astring /opt/oxide/dendrite/misc/port_map.csv
config/vlan_link astring tfportint0_0
config/techport0_prefix astring fdb1:a840:2504:355::
config/techport1_prefix astring fdb2:a840:2504:355::
config/pkt_source astring tfpkt0
config/host astring [::1]
config/port astring 12224
```

And to confirm, this is actually emitting the unsolicited NAs on the right techport VLAN interfaces:

```bash
^CBRM44220001 # zlogin oxz_switch snoop -r -d techport0 -v 'icmp6'
Using device techport0 (promiscuous mode)
ETHER:  ----- Ether Header -----
ETHER:
ETHER:  Packet 1 arrived at 9:08:5.26188
ETHER:  Packet size = 102 bytes
ETHER:  Destination = 33:33:0:0:0:1, (multicast)
ETHER:  Source      = a8:40:25:5:6:2,
ETHER:  Ethertype = 86DD (IPv6)
ETHER:
IPv6:   ----- IPv6 Header -----
IPv6:
IPv6:   Version = 6
IPv6:   Traffic Class = 0
IPv6:   Flow label = 0x0
IPv6:   Payload length = 48
IPv6:   Next Header = 58 (ICMPv6)
IPv6:   Hop Limit = 255
IPv6:   Source address = fe80::aa40:25ff:fe05:602
IPv6:   Destination address = ff02::1
IPv6:
ICMPv6:  ----- ICMPv6 Header -----
ICMPv6:
ICMPv6:  Type = 134 (Router advertisement)
ICMPv6:  Code = 0
ICMPv6:  Checksum = 9603
ICMPv6:  Max hops= 255, Router lifetime= 0
ICMPv6:  Managed addr conf flag: NOT SET, Other conf flag: NOT SET
ICMPv6:  Reachable time: 0, Reachable retrans time 0
ICMPv6:
ICMPv6:  +++ ICMPv6 Prefix option +++
ICMPv6:  Prefix length = 64
ICMPv6:  Onlink flag: SET, Autonomous addr conf flag: SET
ICMPv6:  Valid Lifetime 120, Preferred Lifetime 60
ICMPv6:  Prefix fdb1:a840:2504:355::
ICMPv6:
^^CBRM44220001 # zlogin oxz_switch snoop -r -d techport0 -v 'icmp6'
Using device techport0 (promiscuous mode)
ETHER:  ----- Ether Header -----
ETHER:
ETHER:  Packet 1 arrived at 9:08:5.26188
ETHER:  Packet size = 102 bytes
ETHER:  Destination = 33:33:0:0:0:1, (multicast)
ETHER:  Source      = a8:40:25:5:6:2,
ETHER:  Ethertype = 86DD (IPv6)
ETHER:
IPv6:   ----- IPv6 Header -----
IPv6:
IPv6:   Version = 6
IPv6:   Traffic Class = 0
IPv6:   Flow label = 0x0
IPv6:   Payload length = 48
IPv6:   Next Header = 58 (ICMPv6)
IPv6:   Hop Limit = 255
IPv6:   Source address = fe80::aa40:25ff:fe05:602
IPv6:   Destination address = ff02::1
IPv6:
ICMPv6:  ----- ICMPv6 Header -----
ICMPv6:
ICMPv6:  Type = 134 (Router advertisement)
ICMPv6:  Code = 0
ICMPv6:  Checksum = 9603
ICMPv6:  Max hops= 255, Router lifetime= 0
ICMPv6:  Managed addr conf flag: NOT SET, Other conf flag: NOT SET
ICMPv6:  Reachable time: 0, Reachable retrans time 0
ICMPv6:
ICMPv6:  +++ ICMPv6 Prefix option +++
ICMPv6:  Prefix length = 64
ICMPv6:  Onlink flag: SET, Autonomous addr conf flag: SET
ICMPv6:  Valid Lifetime 120, Preferred Lifetime 60
ICMPv6:  Prefix fdb1:a840:2504:355::
ICMPv6:

^CBRM44220001 # zlogin oxz_switch snoop -r -d techport1 -v 'icmp6'
Using device techport1 (promiscuous mode)
ETHER:  ----- Ether Header -----
ETHER:
ETHER:  Packet 1 arrived at 9:09:5.33908
ETHER:  Packet size = 102 bytes
ETHER:  Destination = 33:33:0:0:0:1, (multicast)
ETHER:  Source      = a8:40:25:5:6:2,
ETHER:  Ethertype = 86DD (IPv6)
ETHER:
IPv6:   ----- IPv6 Header -----
IPv6:
IPv6:   Version = 6
IPv6:   Traffic Class = 0
IPv6:   Flow label = 0x0
IPv6:   Payload length = 48
IPv6:   Next Header = 58 (ICMPv6)
IPv6:   Hop Limit = 255
IPv6:   Source address = fe80::aa40:25ff:fe05:602
IPv6:   Destination address = ff02::1
IPv6:
ICMPv6:  ----- ICMPv6 Header -----
ICMPv6:
ICMPv6:  Type = 134 (Router advertisement)
ICMPv6:  Code = 0
ICMPv6:  Checksum = 9602
ICMPv6:  Max hops= 255, Router lifetime= 0
ICMPv6:  Managed addr conf flag: NOT SET, Other conf flag: NOT SET
ICMPv6:  Reachable time: 0, Reachable retrans time 0
ICMPv6:
ICMPv6:  +++ ICMPv6 Prefix option +++
ICMPv6:  Prefix length = 64
ICMPv6:  Onlink flag: SET, Autonomous addr conf flag: SET
ICMPv6:  Valid Lifetime 120, Preferred Lifetime 60
ICMPv6:  Prefix fdb2:a840:2504:355::
ICMPv6:

```

So we're sending an NA for `fdb1:a840:2504:355::/64` on `techport0` and `fdb2:a840:2504:355::` on `techport1` as expected.

**Review by rcgoodfellow — APPROVED** [review]
LGTM. Thanks!

**Comment by bnaecker** [comment]
Thanks for the look Ry, just waiting on CI which appears like it won't resolve anytime soon. Many runs are in limbo at the moment...

File: package-manifest.toml (hunk 1 above)
Comment (bnaecker) [reply package-manifest.toml:321]: Done in 062e6962

**Pull request merged**